test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -593,8 +593,13 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				By("Updating the performance profile")
 				profiles.UpdateWithRetry(profile)
 
-				By("Applying changes in performance profile and waiting until mcp will start updating")
-				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+				expectedMCPUpdatingStatus := corev1.ConditionTrue
+				if isDefaultWorkloadHints(*initialProfile.Spec.WorkloadHints) {
```

Comment:
can this be `nil`?

Reply:
yes, I'll address this. thanks for pointing it out